.gitignore

```diff
@@ -24,3 +24,6 @@ testbin/*
 *.swo
 *~
 .env
+
+#
+/images/fluentbit/conf/output.conf
```

Comment:
This ignore is not needed, right?

Reply:
This file will be generated when `gen-config.pl` get manually called. Ignoring it would make development easier.